app/models/setting/content.rb

```diff
@@ -14,6 +14,7 @@ def self.default_settings
     download_policies = proc { hashify_parameters(::Runcible::Models::YumImporter::DOWNLOAD_POLICIES) }
     proxy_download_policies = proc { hashify_parameters(::SmartProxy::DOWNLOAD_POLICIES) }
     dependency_solving_options = proc { hashify_parameters(['conservative', 'greedy']) }
+    cdn_ssl_versions = proc { hashify_parameters(['SSLv23', 'TLSv1']) }
```

Comment:
If we go with my suggestion around the SUPPORTED_SSL_VERSIONS constant we can reference if here ie:

```
proc { hashify_parameters(Katello::Resources::CDN::SUPPORTED_SSL_VERSIONS) }
```